### PR TITLE
Implement empty state for packing list (+ starting to implement full functionality)

### LINKED ISSUE #23: Implement packing list page empty state
**User Story**
As a road trip organiser, I want to set a packing list for my road trip, so that the attendees know what to pack.

**Acceptance Criteria**

- Implement basic design of empty state
- Check backend data structure match the information that is needed
- Implement the ability to add item(s) to the packing list

**Additional Context**
Screen Shot 2021-04-13 at 2 34 54 PM

**Dependencies**
N/A

### PR DESCRIPTION
**Pull request by Denijar**
**Changes Made:**

This PR adds a Packing List card to the PackingList page. Currently, it will show an empty message if there is an empty packing list. If the user is the organiser, then the '+ Add Item' button will be visible.

Look when empty and user is the organiser with sidebar:
image

Look when populated and user is not organiser, sidebar collapsed:
image

Closes #23

### CODE REVIEW

File: frontend/src/pages/RoadTripPage/PackingListPage/PackingListPage.jsx [hunk 1]
```diff
@@ -1,9 +1,6 @@
 import { React } from "react";
+import PackingList from "../../../components/PackingList/PackingList";
 
 export default function PackingListPage() {
-  return (
-    <div>
-      <p>Packing List</p>
-    </div>
-  );
+    return <PackingList />;
```
Comment (Denijar) [inline frontend/src/pages/RoadTripPage/PackingListPage/PackingListPage.jsx:5]: So technically everything in the PackingList.jsx file could be in here instead. Personally I like how it separates the two ideas: "this is the page and then this is the element on the page" but it also means this file has only one line of code. Thoughts?

Comment (Jennak21) [reply frontend/src/pages/RoadTripPage/PackingListPage/PackingListPage.jsx:5]: I like this!  If in the future we wanted to show other components on the page, it makes sense to have the page separated from its components.

File: frontend/src/components/PackingList/PackingList.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import styles from "./PackingList.module.css";
+import { Button } from "@material-ui/core";
+import { withStyles } from "@material-ui/core";
+
+import Item from "./Item/Item";
+
+const AddItemButton = withStyles(() => ({
+    root: {
```
Comment (Jennak21) [inline frontend/src/components/PackingList/PackingList.jsx:9]: Is there a way of having this in a css style sheet to make things cleaner

Comment (Denijar) [reply frontend/src/components/PackingList/PackingList.jsx]: good idea, I made `PackingList.styles.js` and `Item.styles.js` files to pull this MUI styling out.

File: frontend/src/pages/RoadTripPage/PackingListPage/PackingListPage.jsx (hunk 1 above)
Comment (lhuijser) [reply frontend/src/pages/RoadTripPage/PackingListPage/PackingListPage.jsx:5]: Ooh interesting, I like this idea! It makes sense to me, might get tricky if we want to nest pages etc inside there, thinking about how I did the map page, but I think this is clean

**Review by lhuijser — APPROVED** [review]
Looks clean as! Love it :D Big approve from me

**Review by courtneyeh — APPROVED** [review]
Love it! Looks real nice!

**Pull request merged**